src/main/java/com/nexym/clinic/domain/doctor/model/Doctor.java

```diff
@@ -0,0 +1,59 @@
+package com.nexym.clinic.domain.doctor.model;
+
+import com.nexym.clinic.domain.availability.model.Availability;
+import com.nexym.clinic.domain.user.model.Civility;
+import com.nexym.clinic.utils.FormatUtil;
+import lombok.Builder;
+import lombok.Data;
+
+import java.time.LocalDateTime;
+import java.util.ArrayList;
+import java.util.List;
+
+@Data
+@Builder
+public class Doctor {
+
+    private Long id;
+    private Long userId;
+    private Long ruleId;
+    private Long specialityId;
+    private Civility civility;
+    private String firstName;
+    private String lastName;
+    private String address;
+    private String email;
+    private String password;
+    private String phoneNumber;
```

Comment:
better to extend User model from doctor and patient in order to reduce duplication